src/stats/boxplot.jl

```diff
@@ -168,11 +176,18 @@ function Makie.plot!(plot::BoxPlot)
     outliers = @lift($signals.outliers)
     t_segments = @lift($signals.t_segments)
     boxwidth = @lift($signals.boxwidth)
+    outlier_indices = @lift($signals.outlier_indices)
+    boxcolor = @lift($signals.boxcolor)
 
-    outliercolor = lift(plot[:outliercolor], plot[:color]) do outliercolor, color
-        outliercolor === automatic || return outliercolor
-        c = to_color(color)
-        return RGB(red(c), green(c), blue(c))
+    outliercolor = lift(plot[:outliercolor], plot[:color], outlier_indices) do outliercolor, color, outlier_indices
+        if outliercolor === automatic
+            color isa AbstractVector || return color
+            return [color[i] for i in outlier_indices]
+        elseif outliercolor isa AbstractVector
+            return outliercolor[outlier_indices]
+        else
+            return outliercolor
```

Comment:
Maybe just start with `c = outliercolor === automatic ? outliercolor : color`, and then work with `c` without having to repeat the logic below.

Reply:
Yes it's prettier, I did it in https://github.com/JuliaPlots/Makie.jl/pull/1032/commits/928f0864fc3a5332171e1a107adb681848e23427